src/util/is-connected.ts

```diff
@@ -0,0 +1,17 @@
+import { Subscription } from "models/subscription";
+import { GitHubServerApp } from "models/github-server-app";
+import { Request } from "express";
+import { getConnectionsAndInstallations } from "routes/jira/jira-get";
```

Comment:
This is a bit strange in terms of importing. `jira-delete` --> import `utils/is-connecrted` --> import `jira-get`.